src/main/resources/reference.conf

```diff
@@ -243,15 +243,15 @@ divolte {
     // the Kafka producer.
     // See: http://kafka.apache.org/documentation.html#producerconfigs
     producer = {
-      metadata.broker.list = ["localhost:9092"]
-      metadata.broker.list = ${?DIVOLTE_KAFKA_BROKER_LIST}
-
+      bootstrap.servers = ["localhost:9092"]
+      bootstrap.servers = ${?DIVOLTE_KAFKA_BROKER_LIST}
       client.id = divolte.collector
       client.id = ${?DIVOLTE_KAFKA_CLIENT_ID}
 
-      request.required.acks = 0
-      message.send.max.retries = 5
-      retry.backoff.ms = 200
+      acks = 1
+      retries = 0
+      compression.type = lz4
+      max.in.flight.requests.per.connection = 1
```

Comment:
Also, I think you want to have a lot more in flight requests per connection as well as retries, given how the existing collector behaves in general.